extension/scripts/global/functions/filters.js

```diff
@@ -148,13 +149,14 @@ function createFilterSection(options) {
 		const rangeSlider = new DualRangeSlider(options.slider);
 		section.appendChild(rangeSlider.slider);
 		section.appendChild(document.newElement({ type: "div", class: "slider-counter", text: "" }));
+		section.classList.add("tt-slider");
 
 		new MutationObserver(options.callback).observe(rangeSlider.slider, { attributes: true });
 
 		return { element: section, getStartEnd, updateCounter };
 
-		function getStartEnd() {
-			return { start: rangeSlider.slider.dataset.low, end: rangeSlider.slider.dataset.high };
+		function getStartEnd(content) {

```

Comment:
Why do you keep asking for the content when you already have the sliders available through `rangeSlider.slider`?